packages/toast/test/standalone-toast.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect } from "react"
+import { render } from "@chakra-ui/test-utils"
+import { createStandaloneToast } from "../src"
+import theme from "@chakra-ui/theme"
```

Comment:
Make sure to add `@chakra-ui/theme` to `devDependencies` in `packages/toast/package.json`

Reply:
✅  done